src/javascripts/components/airports.js

```diff
@@ -0,0 +1,23 @@
+const showAirports = (array) => {
+  document.querySelector('#boards').innerHTML = '';
+  document.querySelector('#add-button').innerHTML = '<button class="btn btn-success btn-lg mb-4" id="add-airport-btn">Add An Airport</button>';
+  document.querySelector('#main-container').innerHTML = '';
+
+  array.forEach((item) => {
+    document.querySelector('#main-container').innerHTML += `<div class="card">
+    <img src='${item.airportImage}' class="card-img-top" alt="...">
+    <div class="card-body">
+    <h5 id="airport-name-title--${item.firebaseKey}" class="card-title">${item.airportName}</h5>
+      <p class="card-desc">${item.airportLocation}</p>
+      <a href="#" class="btn btn-danger" id="editAirport--${item.firebaseKey}">Edit Airport</a>
```

Comment:
Looks like your edit buttons and delete buttons are showing up even if you're logged out. We want them to only show up if you're logged in. We can add this functionality later. Create a ticket to pull in at the end of this sprint and add a link to it here as a response! If you already have a ticket for this work you can add that here instead!

Reply:
We have a workaround in the next pull. I'll make a note and double-check this is what is needed.

Reply:
Yeah! Can you make an issue to track that work? Also so that I can remember to check for it in the next Milestone approval.